### ISSUE TITLE
Windows/Windows Phone/Windows Universal confusion

### ISSUE THREAD

**Issue by brody2consult** [jessemonroy650/top-phonegap-mistakes#8]
Microsoft introduced the "Windows Universal" designation to mean Windows 8, Windows 8.1, or Windows Phone 8.1 then deprecated due to the coming of the UWP (Universal Windows Platform). Also, the Cordova `windows` target added UWP (Windows 10) and dropped Windows 8 (which is supported by the Cordova `windows8` target). Another thing is that it is possible for a someone to develop a Windows Phone 8.1 app using either the `windows` target or the `wp8` target.

I think it would be very easy for a new developer to get confused by all of these ideas.

**Comment by jessemonroy650** [comment]
> I think it would be very easy for a new developer to get confused by all of these ideas.

I agree, but I don't run the Microsoft marketing machine. Is there a drawing somewhere to explain all this? Do you have one? or should I draw one?

**Comment by brody2consult** [comment]
I will raise a PR when I get a chance (I am on vacation and am busy with a critical issue ~~with~~ _for_ someone).

Windows Universal used to mean separate builds of the same project on Windows 8, Windows 8.1, and Windows Phone 8.1, as documented by cordova-windows in August 2014: https://github.com/apache/cordova-windows/blob/97a07e24d65f47c30558199787cda6c5ca0192f5/README.md

_EDIT:_ also documented by Telerik at: http://www.telerik.com/windows-universal-ui

Now the "Universal Windows Platform" (UWP) means a single build for Windows 10 that runs on all Windows 10 devices, such as desktops, laptops, mobile, and _maybe_ IoT. This is very well documented on the internet, for example (with a nice image): https://msdn.microsoft.com/en-us/library/windows/apps/dn894631.aspx

**Comment by jessemonroy650** [comment]
Okay. I'll make some accomidation for it. Can you poke me when you get back from vacation. Right now I'm gonna finish up the whitelist stuff, then create a large "demo" set of Applications that use the "core" plugins. It will be one App for each "core" plugin. It is intended for demos, as Cordova/Phonegap recent dropped this from the documentation.

**Comment by brody2consult** [comment]
Will do, I will be back on January 6th (poke me in case I forget). I would also like to thank you for your support in the JIRA reports and Nitobi/PhoneGap forum related to #7.
